src/recon.rs

```diff
@@ -1395,22 +1396,23 @@ fn read_coef_tree<BD: BitDepth>(
                     ytx, txtp, eob, ts_c.msac.rng,
                 );
             }
-            CaseSet::<16, true>::many(
-                [&t.l.lcoef, &f.a[t.a].lcoef],
-                [
+            case_set_default!(
+                16,
+                dir = [&t.l.lcoef, &f.a[t.a].lcoef],
```

Comment:
```suggestion
                dir = [&t.l, &f.a[t.a]],
```

I did this change in my version, since I think it's something I accidentally did during initial safety cleanup.  Same for the couple of other places this happens.